### PR TITLE
Fix integration tests for mysql and postgres

### PR DESCRIPTION
**Pull request by idafurjes**
**What this PR does / why we need it**:
We moved away from using integration tags on #49448. The grabpl commands in postgres and mysql integration tests were not running any tests due to the change. Here we replace the grabpl commands with go test commands running the tests for both mysql and postgres.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -454,15 +454,30 @@ steps:
...
+  image: grafana/build-container:1.5.5
+  name: wire-install
 - commands:
   - apt-get update
   - apt-get install -yq postgresql-client
   - dockerize -wait tcp://postgres:5432 -timeout 120s
   - psql -p 5432 -h postgres -U grafanatest -d grafanatest -f devenv/docker/blocks/postgres_tests/setup.sql
   - go clean -testcache
-  - ./bin/grabpl integration-tests --database postgres
+  - go list './pkg/...' | xargs -I {} sh -c 'GRAFANA_TEST_DB=postgres go test -run
```
Comment (kminehart) [inline .drone.yml:477]: Is the go list ~ xargs necessary? Integration tests for sqlite are ran with `go test -run=Integration ./pkg/...`.

**Review by kminehart — CHANGES_REQUESTED** [review]

File: .drone.yml (hunk 1 above)
Comment (idafurjes) [reply .drone.yml:477]: not running the packages separately makes the test fail, so we need the go list

Comment (kminehart) [reply .drone.yml:477]: ok!

**Review by kminehart — APPROVED** [review]

File: .drone.yml (hunk 1 above)
Comment (sakjur) [reply .drone.yml:477]: Globals 🎉🥳🎈

**Pull request merged**